tools/tests/systemtests/Systemtest.py

```diff
@@ -354,13 +354,12 @@ def _run_field_compare(self):
                 stdout, stderr = process.communicate(timeout=GLOBAL_TIMEOUT)
             except KeyboardInterrupt as k:
                 process.kill()
-                # process.send_signal(9)
                 raise KeyboardInterrupt from k
             except Exception as e:
                 logging.critical(
-                    f"Systemtest {self} had serious issues executin the docker compose command about to kill the docker compose command. Please check the logs! {e}")
+                    f"Systemtest {self} had serious issues executing the docker compose command about to kill the docker compose command. Please check the logs! {e}")
                 process.kill()
-                stdout, stderr = process.communicate()
+                process.communicate(timeout=10)
```

Comment:
What is this magic 10? Should it be:

```suggestion
                process.communicate(timeout=GLOBAL_TIMEOUT)
```
?

Why are the `stdout, stderr` not needed here?

I understand that this is a timeout just for trying to launch Docker. Maybe a comment here would help distinguish from the rest.